src/SeleniumLibrary/__init__.py

```diff
@@ -384,11 +385,22 @@ def failure_occurred(self):
             self._running_on_failure_keyword = False
 
     @property
+    def driver(self):
+        """Current active driver.
+
+        :rtype: selenium.webdriver.remote.webdriver.WebDriver
+        :raises RuntimeError if driver is not created
+        """
+        if not self._drivers.current:
+            raise NoOpenBrowser('No browser is open')
```

Comment:
- Exception type not updated in the doc string above.
- This error message could have `.` at the end.

Reply:
Done.